README.md

```diff
@@ -1,2 +1,74 @@
 # StaticMaps
-Static Map renderer using mapnik
+Static Map rendering service using mapnik. Post GeoJSON, get back a map rendered on a background.
+Written in python, using django.
+Designed to be used as a micro service and deployed as a docker container.
+
+Prebuilt docker images available as gaiagps/staticmapservice, If you want to add your own background, extend that image. 
+
+For an example service file for deploying with fleet see staticmaps@.service
+
+## API
```

Comment:
I've always wondered if there is a standard for documenting APIs, and is this it?